### PR TITLE
RNs-4.7.51: adds z-stream RNs

### PR DESCRIPTION
**Pull request by JoeAldinger**
RNs for 4.7.51 z-stream

Version(s):
4.7

Issue:
[Jira](https://issues.redhat.com/browse/OCPPLAN-9105)

Link to docs preview:
[Preview](https://joealdinger.github.io/openshift-docs/RNs-4.7.51/release_notes/ocp-4-7-release-notes.html#ocp-4-7-51)

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -3292,3 +3292,27 @@ link:https://access.redhat.com/solutions/6957908[{product-title} 4.7.50 containe
...
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6959299[{product-title} 4.7.51 container image list]
+
+[id="ocp-4-7-51-features"]
+==== Features
+
+[id="ocp-kubernetes-1-21-11-updates"]
+===== Updates from Kubernetes 1.21.11
```
Comment (opayne1) [inline release_notes/ocp-4-7-release-notes.adoc:3311]: This should be 1.20.14 based on what I saw in the bug.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -3292,3 +3292,27 @@ link:https://access.redhat.com/solutions/6957908[{product-title} 4.7.50 containe
...
+
+This update contains changes from Kubernetes 1.21.9 up to 1.21.11. More information can be found in the following changelogs: link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#downloads-for-v1219[1.21.9],link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#downloads-for-v12110[1.21.10], and link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#downloads-for-v12111[1.21.11].
```
Comment (opayne1) [inline release_notes/ocp-4-7-release-notes.adoc:3313]:
```suggestion
This update contains changes from Kubernetes 1.20.14 up to 1.20.15. More information can be found in the following changelog: link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#v12015[1.20.15].
```

**Review by opayne1 — COMMENTED** [review]
See https://bugzilla.redhat.com/show_bug.cgi?id=2082029. Kubernetes updates are from 1.20.14 to 1.20.15.

**Comment by bergerhoffer** [comment]
LGTM too, merging per slack request!

**Pull request merged**